### ISSUE TITLE
abnormal muscle triglyceride level

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3599]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

a. abnormal skeletal muscle triglyceride level
b. increased skeletal muscle triglyceride level
c. decreased skeletal muscle triglyceride level

>2. Synonyms (e.g., absent spleen)

a. abnormal muscle triglyceride level
b. increased muscle triglyceride level
c. decreased muscle triglyceride level; reduced muscle triglyceride level; reduced skeletal muscle triglyceride level

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

a.  aberrant concentration of naturally occurring esters of three fatty acids and glycerol in skeletal muscle; triglycerides are widespread in adipose tissue, commonly circulate in the blood carried by lipoproteins, and are involved in the process of bidirectional transference of adipose fat and blood glucose
b.  greater concentration of naturally occurring esters of three fatty acids and glycerol in skeletal muscle; triglycerides are widespread in adipose tissue, commonly circulate in the blood carried by lipoproteins, and are involved in the process of bidirectional transference of adipose fat and blood glucose
c.  reduced concentration of naturally occurring esters of three fatty acids and glycerol in skeletal muscle; triglycerides are widespread in adipose tissue, commonly circulate in the blood carried by lipoproteins, and are involved in the process of bidirectional transference of adipose fat and blood glucose

PMID:11254461

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

a.  abnormal triglyceride level (MP:0000187)
b.  abnormal skeletal muscle triglyceride level
c.  abnormal skeletal muscle triglyceride level

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5256-8683

**Comment by anna-anagnostop** [comment]
A review of MP annotations to abnormal triglyceride level MP:0000187 indicates that we need a new term for abnormal cardiac muscle triglyceride level in addition to a new term for abnormal skeletal muscle triglyceride level. Hence, abnormal muscle triglyceride level and abnormal skeletal muscle triglyceride level will NOT be used synonymously.  Instead, abnormal muscle triglyceride level will be used as a grouping (parent) term for the cardiac muscle and skeletal muscle child terms. See abnormal muscle glycogen level [MP:0014068 ](http://www.informatics.jax.org/vocab/mp_ontology/MP:0014068)

abnormal muscle triglyceride level (a child of abnormal triglyceride level MP:0000187)
DEF: any anomaly in the concentration of naturally occurring esters of three fatty acids and glycerol in muscle tissue

abnormal skeletal muscle triglyceride level
DEF: any anomaly in the concentration of naturally occurring esters of three fatty acids and glycerol in skeletal muscle tissue

decreased skeletal muscle triglyceride level
DEF: less than the normal concentration of naturally occurring esters of three fatty acids and glycerol in skeletal muscle tissue
SYN: reduced skeletal muscle triglyceride level

increased skeletal muscle triglyceride level
DEF: greater than the normal concentration of naturally occurring esters of three fatty acids and glycerol in skeletal muscle tissue
SYN: elevated skeletal muscle triglyceride level

abnormal cardiac muscle triglyceride level
DEF: any anomaly in the concentration of naturally occurring esters of three fatty acids and glycerol in heart muscle 
SYN: abnormal heart muscle triglyceride level
SYN: abnormal myocardial triglyceride level
SYN: abnormal cardiomyocyte triglyceride level

decreased cardiac muscle triglyceride level
DEF: less than the normal concentration of naturally occurring esters of three fatty acids and glycerol in heart muscle
SYN: reduced heart muscle triglyceride level
SYN: reduced cardiac muscle triglyceride level
SYN: decreased myocardial triglyceride level
SYN: reduced myocardial triglyceride level
SYN: decreased cardiomyocyte triglyceride level
SYN: reduced cardiomyocyte triglyceride level

increased cardiac muscle triglyceride level
DEF: greater than the normal concentration of naturally occurring esters of three fatty acids and glycerol in heart muscle
SYN: elevated cardiac muscle triglyceride level
SYN: increased heart muscle triglyceride level
SYN: increased cardiomyocyte triglyceride level
SYN: increased myocardial triglyceride level

**Comment by anna-anagnostop** [comment]
Added new terms:

abnormal muscle triglyceride level MP:0031412
abnormal skeletal muscle triglyceride level MP:0031413
decreased skeletal muscle triglyceride level MP:0031414
increased skeletal muscle triglyceride level MP:0031415
abnormal cardiac muscle triglyceride level MP:0031416
decreased cardiac muscle triglyceride level MP:0031417
increased cardiac muscle triglyceride level MP:0031418
